CHANGES.rst

```diff
@@ -7,6 +7,7 @@ Changes
 - Dropped support for Django<=1.10 (gh-356)
 - Fix bug where history_view ignored user permissions (gh-361)
 - Fixed HistoryRequestMiddleware which hadn't been working for Django>1.9 (gh-364)
+- Add ability to specify custom history_id field (gh-368)
```

Comment:
This isn't part of the 2.0 release. So you can add another section above like this:
```
Unreleased
------------
- Add ability to specify custom history_id field (gh-368)
```